### PR TITLE
Use app store to get user details instead of cookie #1976

### LINKED ISSUE #1976: Use app store to get user details instead of cookie
**Actual Behaviour**

In some component, cookie is used instead of redux store

**Expected Behaviour**

Use redux store to get user-related data instead of relying on cookies

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1976 

Changes:

- Used app store to get user details
- Connected redux with NotFound, Support
- Added Tests for NotFound, Support

Demo Link: https://pr-1977-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
Ignore codacy error

File: src/components/NotFound/NotFound.react.js [hunk 1]
```diff
@@ -54,7 +57,8 @@ export default class NotFound extends Component {
   };
   // Open Login Dialog
   handleLoginOpen = () => {
-    if (cookies.get('loggedIn')) {
+    const { accessToken } = this.props;
+    if (accessToken) {
       window.location = urls.CHAT_URL;
```
Comment (akshatnitd) [inline src/components/NotFound/NotFound.react.js]: This will cause reload. Use `this.props.history.replace`

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Comment by fragm3** [comment]
@akshatnitd made the changes

**Comment by fragm3** [comment]
Ignore codacy

**Pull request merged**